generator/generate_md.py

```diff
@@ -0,0 +1,65 @@
+class MdFile:
```

Comment:
Prefix must be one of the following ['LEETCODE', 'GENERATOR', 'TRIANGLES', 'HEXNUMBER', 'REQUESTS', 'ITERATOR'].
Group must be one of the following ['1021', '1022'].